deblur/deblurring.py

```diff
@@ -52,7 +52,7 @@ def get_sequences(input_seqs):
     return seqs
 
 
-def deblur(input_seqs, read_error=0.05, mean_error=None, error_dist=None,
+def deblur(input_seqs, read_error=0.05, mean_error=0.005, error_dist=None,
```

Comment:
@amnona can you explain the rationale behind this change? I though that if the user doesn't provide a `mean_error` we were gonna use the same value as `read_error`. Did that change based on recent benchmarks?